src/scipp/html/templates/style.css.template

```diff
@@ -19,6 +19,20 @@
   --sc-table-header-font-color: $header_text_color;
 }
 
+html[theme=dark],
```

Comment:
This also selects non-vs-code elements. Did you check that this doesn't affect Jupyter notebook / lab?
Also, should we even apply these variables outside of `.sc-root`?

Reply:
You're right. It's better to only keep the last path.